### PR TITLE
ContactForm Widget

### PR DESCRIPTION
**Pull request by krystian15**
Simple contact form for ScandiPWA
TODO:
Fix form clear after sent @alfredsgenkins, we are talking about it (Slack) :D  
@pwalus

### CODE REVIEW

File: src/app/component/ContactForm/ContactForm.style.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+.ContactForm {}
```
Comment (Danpiel) [inline src/app/component/ContactForm/ContactForm.style.scss]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/app/query/ContactForm.query.js  1
- src/app/component/FieldForm/FieldForm.component.js  1
- src/app/store/ContactForm/ContactForm.reducer.js  2
- src/app/component/ContactForm/ContactForm.component.js  3
- src/app/store/ContactForm/ContactForm.dispatcher.js  1
- src/app/store/ContactForm/ContactForm.action.js  1
- src/app/component/ContactForm/ContactForm.container.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886&bid=16018264)

File: src/app/component/ContactForm/ContactForm.style.scss (hunk 1 above)
Comment (Danpiel) [inline src/app/component/ContactForm/ContactForm.style.scss]: Codacy Issue found: [Unexpected empty block (block-no-empty)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/app/component/ContactForm/ContactForm.container.js  1
- src/app/component/FieldForm/FieldForm.component.js  1
- src/app/store/ContactForm/ContactForm.reducer.js  2
- src/app/store/ContactForm/ContactForm.action.js  1
- src/app/store/ContactForm/ContactForm.dispatcher.js  1
- src/app/component/ContactForm/ContactForm.component.js  3
- src/app/query/ContactForm.query.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886&bid=16018264)

File: src/app/component/ContactForm/ContactForm.style.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Danpiel) [inline src/app/component/ContactForm/ContactForm.style.scss]: Codacy Issue found: [Unexpected empty source (no-empty-source)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886)

**Comment by alfredsgenkins** [comment]
Hey!

Outstanding code quality! Applauds!

However, it seems like our automated PR review tool is not doing it's job properly - we have a .eslint file - try enabling it (renaming from .sample). If you have es-lint as an extension - you are good to go.

I will take a look on this soon, if you will fix the real es-lint issue - I would be grateful.

Thanks again for amazing contribution!

**Comment by krystian15** [comment]
> Hey!
> 
> Outstanding code quality! Applauds!
> 
> However, it seems like our automated PR review tool is not doing it's job properly - we have a .eslint file - try enabling it (renaming from .sample). If you have es-lint as an extension - you are good to go.
> 
> I will take a look on this soon, if you will fix the real es-lint issue - I would be grateful.
> 
> Thanks again for amazing contribution!

@alfredsgenkins I fixed this files code standard,
module for this component you can find in your slack priv message :D

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/app/component/FieldForm/FieldForm.component.js  1
- src/app/component/ContactForm/ContactForm.component.js  3
- src/app/query/ContactForm.query.js  1
- src/app/store/ContactForm/ContactForm.action.js  1
- src/app/store/ContactForm/ContactForm.reducer.js  2
- src/app/store/ContactForm/ContactForm.dispatcher.js  1
- src/app/component/ContactForm/ContactForm.container.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4816886&bid=16018264)

**Comment by alfredsgenkins** [comment]
@all-contributors please add @pwalus for code!

**Comment by alfredsgenkins** [comment]
This issue is blocked for the reason: it is preferable to revisit after we launch plugin functionality.

**Comment by yeegor** [comment]
I guess it is time to revisit this! 
@alfredsgenkins please take a look once again.

**Comment by alfredsgenkins** [comment]
I think due inactivity we will close this issue. The contact for will soon appear in ScandiPWA.

**Pull request closed without merging**